recipes/stages/_base_/data/pipelines/seg_semisl.py

```diff
@@ -0,0 +1,56 @@
+__img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+__img_scale = (544, 544)
+__crop_size = (512, 512)
+
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='LoadAnnotations'),
+    dict(type='Resize', img_scale=__img_scale, ratio_range=(0.5, 2.0), keep_ratio=False),
+    dict(type='RandomCrop', crop_size=__crop_size, cat_max_ratio=0.75),
+    dict(type='RandomFlip', prob=0.5, direction='horizontal'),
+    dict(type='RandomRotate', prob=0.5, degree=30, pad_val=0, seg_pad_val=255),
+    dict(type='Normalize', **__img_norm_cfg),
+    dict(type='Pad', size=__crop_size, pad_val=0, seg_pad_val=255),
+    dict(type='DefaultFormatBundle'),
+    dict(type='Collect', keys=['img', 'gt_semantic_seg'])
+]
+test_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(
+        type='MultiScaleFlipAug',
+        img_scale=__img_scale,
+        flip=False,
+        transforms=[
+            dict(type='Resize', keep_ratio=False),
+            dict(type='RandomFlip'),
+            dict(type='Normalize', **__img_norm_cfg),
+            dict(type='ImageToTensor', keys=['img']),
+            dict(type='Collect', keys=['img'])
+        ]
+    )
+]
+
+unlabeled_pipeline = [
```

Comment:
There are a lot of code duplication. Could you use `append` for new augs to train_pipeline list?